yarn.lock

```diff
@@ -725,6 +725,13 @@ debug@^3.1.0:
   dependencies:
     ms "2.0.0"
 
+debug@^4.1.1:
```

Comment:
I would have expected this dependency to replace the 3.1.0 dep above?

Reply:
The dev dependencies `istanbul-lib-source-maps` and `lint-staged` depend on debug 3.1, thus the occurrence in the lock file.